src/server/schemas/wallet-subscription.ts

```diff
@@ -0,0 +1,56 @@
+import { Type } from "@sinclair/typebox";
+import type { Prisma, Webhooks } from "@prisma/client";
+import { AddressSchema } from "./address";
+import {
+  WalletConditionsSchema,
+  validateConditions,
+} from "../../shared/schemas/wallet-subscription-conditions";
+
+interface WalletSubscriptionWithWebhook {
```

Comment:
We shouldn't need to define a different interface right? Shouldn't we be passing the type
`import type { WalletSubscription } from "@prisma/client"` around?

Or if we intend to pass the type joined with Webhook, then something like:
```typescript
type WalletSubscriptionWithWebhook = WalletSubscriptions & { webhook: Webhooks }
```
```